### PR TITLE
Change Check Client command format

### PR DESCRIPTION
**Pull request by FeliciaBeatrice**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -213,9 +213,24 @@ List only selected details of all the clients, depending on TAG. The commands fo
...
+  Contact Number: 93437878
+  Email: doja88@example.com
+  Budget: SGD2000/month
+
+Here is the property this client is renting:
+  Landlord: Bob Tan Bee Bee
+  Address: 25 Lower Kent Ridge Rd, S119081
+  Renting Price: SGD1000/month
+  Unit Type: LP Bungalow
+```
```
Comment (wilsonngja) [inline docs/UserGuide.md:233]: This is a nice addition to the user guide. Great job

File: src/main/java/seedu/duke/parsermanager/ParseCheckClient.java [hunk 2]
```diff
@@ -45,19 +50,28 @@ public Command parseCommand() throws InvalidIndexException, MissingFlagException
...
         String[] flags = CHECK_CLIENT_FLAGS;
         int[] flagIndexPositions = getFlagIndexPositions(rawCommandDetail, flags);
+        checkForExtraFlags(flagIndexPositions);
         checkForMissingFlags(flagIndexPositions);
         checkFlagsOrder(flagIndexPositions);
         return extractCommandDetails(rawCommandDetail, flags, flagIndexPositions);
     }
 
+    private void checkForExtraFlags(int[] flagIndexPositions) throws ExtraFlagsException {
+        if (flagIndexPositions[0] != CORRECT_FLAG_POSITION) {
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/parsermanager/ParseCheckClient.java:70]: Perhaps can use some final string to represent the 0 instead?

**Review by wilsonngja — COMMENTED** [review]
Great effort in adding the expected output. LGTM

**Review by ngdeqi — APPROVED** [review]
LGTM, try to avoid magic literals!

**Pull request merged**